src/components/InstanceMonitoring/InstanceMonitoringCard.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'carbon-components-react';
+import { iconChevronUp } from 'carbon-icons';
+
+export default class InstanceMonitoringCard extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    hideSummaryText: PropTypes.string.isRequired,
+  };
+
+  static defaultProps = {
+    hideSummaryText: 'Hide Summary',
+  };
+
+  render() {
```

Comment:
It appears that this class still doesn’t have any state. Does it make sense to convert this class to stateless?